web/src/components/feature/MoleTable.tsx

```diff
@@ -0,0 +1,51 @@
+import { DynamicTable, TDynamicTableInput } from '../common/DynamicTable'
+import { TComponentResponse } from '../../pages/Main'
+
+// TODO: Update with type from generated api
+type TMultiFlashResponse = {
+  component_fractions: { [key: number]: number[] }
+  phase_values: { [key: string]: { percentage: number; mercury: number } }
```

Comment:
We don't do snake case in typescript 📘

Reply:
this structure represents the API response as it is

Reply:
Not anymore!

Reply:
Right. This should be updated when the code is updated to do an actual API call